swan-lake/development-tutorials/configurability/provide-values-to-configurable-variables.md

```diff
@@ -1,67 +1,137 @@
-| xml            | <code>configurable xml book = ?; </code>                                                                                                                                              | `bal run -- -CxmlVar="<book>The Lost World</book>"`               |
-| enum           | <code>enum Country { </code><br>    <code>LK = "Sri Lanka", </code><br>    <code>US = "United States" </code><br> <code>} </code><br> <code>configurable Country country = ?; </code> | `bal run -- -Ccountry="Sri Lanka"`                                |
-| union          | <code>configurable float&#124;int&#124;string measurement = ?; </code>                                                                                                                | `bal run -- -Cmeasurement=5.0`                                    |
+| boolean        | <code>configurable boolean isAdmin = ?; </code>                                                                                                                                       | `$ bal run -- -CisAdmin=true` <br> or <br> `$ bal run -- -CisAdmin=1` |
+| int, byte      | <code>configurable byte age = ?; </code><br> <code>configurable int port = ?;</code>                                                                                                  | `$ bal run -- -Cage=25 -Cport=9090`                                 |
+| float, decimal | <code>configurable float height = ?; </code><br> <code>configurable decimal salary = ?;</code>                                                                                        | `$ bal run -- -Cheight=5.6 -Csalary=50500.65`                       |
+| string         | <code>configurable string name = ?; </code>                                                                                                                                           | `$ bal run -- -Cname=John`                                          |
+| xml            | <code>configurable xml book = ?; </code>                                                                                                                                              | `$ bal run -- -CxmlVar="<book>The Lost World</book>"`               |
+| enum           | <code>enum Country { </code><br>    <code>LK = "Sri Lanka", </code><br>    <code>US = "United States" </code><br> <code>} </code><br> <code>configurable Country country = ?; </code> | `$ bal run -- -Ccountry="Sri Lanka"`                                |
+| union          | <code>configurable float&#124;int&#124;string measurement = ?; </code>                                                                                                                | `$ bal run -- -Cmeasurement=5.0`                                    |
+
+## Provide via configuration files
+
+You can provide configuration values via one or more configuration files as follows.
```

Comment:
Shall we make this comply with these rules?
https://github.com/ballerina-platform/ballerina-spec/blob/master/configurable/spec.md#locating-toml-files
The given description seems to be a bit deviant from this.

Reply:
Updated.